WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -23,6 +23,11 @@ final class WooShippingCreateLabelsViewModel: ObservableObject {
         shippingLabel != nil
     }
 
+    /// Whether the custom information is completed or not
+    var customsInformationIsCompleted: Bool {
+        true
```

Comment:
This is temporary always true?

Reply:
Exactly, temporarily, added comment here https://github.com/woocommerce/woocommerce-ios/pull/14729/commits/2e93bb36806d4f2c9a774a58585d8b4af63cb0f7